examples/08-vrm/plot_inv_vrm_eq.py

```diff
@@ -196,7 +196,7 @@
 w = w / np.max(w)
 w = w
 
-reg = regularization.Smallness(mesh=mesh, active_cells=actCells, cell_weights=w)
+reg = regularization.Smallness(mesh=mesh, active_cells=actCells, weights=w)
```

Comment:
since this is an example, we probably want to do something like:
```suggestion
reg = regularization.Smallness(mesh=mesh, active_cells=actCells, weights={'cell_weights': w})
```

Reply:
You're right. I was leaving these changes for #1344. Does it make sense?

Reply:
It looks like that PR is tackling something else to me. It's changing the regularization to no longer accepting weights as an array input.

This one is no longer accepting `cell_weights` on initialization. So the logical change is to update it to the long-term correct way of doing it.

Reply:
Yes, this change is related to this PR and to #1344 as well. Right now the code in `main` (and in this PR) still works if the passed `weights` is an array. That's why I thought is not urgent to change it in this PR, but it will be necessary to update all examples and tests in #1344 (specially if we add a check that raises an error if `weights` is not a dict).

I can still apply this change here, no bit deal. But I wouldn't update all examples and tests passing `weights` as arrays in this PR.